cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils import testgen
+
+pytestmark = [
+    test_requirements.quota,
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('uses_infra_providers')
+]
+
+
+def pytest_generate_tests(metafunc):
```

Comment:
**REQUIRED:** Please can you use the new provider marker here as it's a new test and very simple usage
```@pytest.mark.provider([VMwareProvider], scope="module")``` should do it

Reply:
Changed.